src/libraries/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEntry.Unix.cs

```diff
@@ -99,6 +112,8 @@ private ReadOnlySpan<char> FullPath
             }
         }
 
+        private bool HasHiddenPrefix => (_directoryEntry.NameLength > 0 && _directoryEntry.Name[0] == '.');
```

Comment:
minor nit: I am not sure what the official repo guidelines are, but we should try to keep the private properties below the public ones: https://stackoverflow.com/a/310967/5852046
```suggestion
        private bool HasHiddenPrefix => _directoryEntry.NameLength > 0 && _directoryEntry.Name[0] == '.';
```

Reply:
Sure, I can do that. My intention was to keep everything alphabetically sorted by member type, ignoring accessibility.

Reply:
We could use `string.StartsWIth('.')`.